Brokerages/Bitfinex/BitfinexBrokerage.Utility.cs

```diff
@@ -210,6 +210,18 @@ private Holding ConvertHolding(Messages.Position position)
                 CurrencySymbol = "$",
                 Type = SecurityType.Crypto
             };
+
+            try
+            {
+                var tick = GetTick(holding.Symbol);
+                holding.MarketPrice = tick.Value;
+            }
+            catch (Exception exception)
+            {
+                Log.Trace($"BitfinexBrokerage.ConvertHolding(): failed to set {holding.Symbol} market price: {exception}");
```

Comment:
Just wondering if this should be a fatal error, 
failure to get the current price of a holding seems quite serious 🤔

Reply:
Agree let's make it fatal. The alternative would be massive jumps in portfolio value by defaulting to 0 causing lots of risk in strategy.

Reply:
Good idea, addressed 😉